OSPaymentsLib/Models/OSPMTServiceProviderInfoModel.swift

```diff
@@ -0,0 +1,70 @@
+/// Enum indicating the payment processing status.
+enum OSPMTProcessStatus: String {
+    case pending = "pending"
```

Comment:
"pending" or "requires_confirmation"?
I get "requires_confirmation" when testing with confirm = false

Reply:
You get that on Stripe's dashboard, right?

Reply:
I'm asking because this corresponds to the response from our backend. And since @alexgerardojacinto approved it, I imagine a similar approach was followed on Android, right?

Reply:
On the API return of our backend. For status that aren't success or failure it just brings the Stripe status

But this status is returned when requesting payments with confirm = false, which Alexandre said we aren't doing by now.

Reply:
I guess the issue is on the backend server, not on the code. The Stripe status should not be returned by the backend.

Reply:
We just changed the backend implementation. So the issue you arose should not happen anymore.